english/explore-outside/journey-to-flaming-tower/start-running/keep-running/keep-running.md

```diff
@@ -2,3 +2,9 @@ You keep running toward the flaming tower.
 You run until your legs burn and your lungs feel like they will explode.  The presence behind you is getting closer.
 Closer with every step.  You feel hot breath on the back of your neck.
 
+A faint shadow emerges from between your feet. Warm gooey chocolate drips down your neck. "The tower! It's collapsing! It's melting chocolate!" you thought. Before you turn your attention back to your feet, your ankles are under thick chocolate goo.
+Just when you thought the shadow came from the melting tower of chocolate, there is it again. The hot breath on the back of your neck.
+
+Do you: 
+
+[Lay down and wait for the chocolate to dry.](../../../../../light-fire/sprinkler/basement-swim/dried-chocolate/dried-chocolate.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.